README.md

```diff
@@ -256,20 +256,24 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 // times out, requiring another attempt at authentication.
                 "state_timeout_in_seconds": 600,
 
-                // The name of the OIDC claim which provides the mapping of an
-                // OIDC user to an iRODS user account.
-                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
-                "irods_user_claim": "irods_username",
-
-                // The mapping of a user to the provided values. All values must
-                // be matched to map an OIDC user to an iRODS user account.
-                // "irods_user_claim" and "user_attribute_mapping" cannot be used together.
-                "user_attribute_mapping": {
-                    "irods_username": {
-                        "sub": "123-abc-456-xyz",
-                        "email": "rods_user@example.org"
+                // User mapping plugins replace 'user_attribute_mapping' and
+                // 'irods_user_claim'. Addtional plugins may be developed to
+                // suit specific needs.
+                "user_mapping": {
+                    // Full path to plugin to load.
+                    // Currently we supply the following plugins:
+                    //   - local_file
+                    //   - user_claim
+                    "plugin_path": "/usr/lib/irods_http_api/plugins/user_mapping/libirods_http_plugin-local_file.so",
+
+                    // Configuration is plugin specific.
+                    // See your desired plugin's documentation for more details.
+                    // Following is an example of the 'local_file' plugin
+                    // configuration.
+                    "configuration": {
+                        "file_path": "/path/to/file"
                     }
```

Comment:
Should we have a specific example?

Reply:
a local file.... perhaps `/etc/irods_http_api/users.json` as an example?

Reply:
I think the use of `/etc/<anything>` could lead the admin to think the HTTP API relies on `/etc`.

Alternatively, you could use `/path/to/file.json`. The important bit is that users understand the file must contain JSON and have a specific structure.

Do we document the expected JSON structure anywhere?

Reply:
Whoops, good point on the `.json` extension from the both of you. 

> a local file.... perhaps `/etc/irods_http_api/users.json` as an example?

I'm not sure if you meant to also provide an example `users.json`. Think that might be a bit too much.
For the path bit, I agree with Kory.

> I think the use of `/etc/<anything>` could lead the admin to think the HTTP API relies on `/etc`.

I agree. Any 'real' looking path might lead users astray.
 
> ...
> Do we document the expected JSON structure anywhere?

Relevant: https://github.com/irods/irods_client_http_api/pull/331#discussion_r1759094871

Reply:
Done.